apache/superset
# chore(Accessibility): Fix accessibility for 'Show x entries' dropdown in tables

### SUMMARY

This PR addresses an accessibility issue in the table charts component of Apache Superset. The 'Show x entries' dropdown lacked proper form control labeling, which is essential for screen reader users. The label has been updated to ensure that it provides a clear and descriptive indication of its purpose, improving the overall accessibility of the feature. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

![Per page superset](https://github.com/user-attachments/assets/c02d7d35-c6e4-412d-8c07-187a75398e3a)


### TESTING INSTRUCTIONS
1. Navigate to a table chart in Apache Superset.
2. Locate the 'Show x entries' dropdown.
3. Use a screen reader (e.g., NVDA or VoiceOver) to navigate to the dropdown.
4. Verify that the screen reader announces the label correctly as "Show [number] entries," where [number] is the selected value.

### ADDITIONAL INFORMATION


- [x] Has associated issue: Fixes #30669
